WellnessLiving/Wl/Location/View/ViewModel.php

```diff
@@ -130,6 +130,14 @@ class ViewModel extends WlModelAbstract
    */
   public $k_business = '0';
 
+  /**
+   * Key of the business type this location belongs to.
+   *
+   * @get result
+   * @var string
+   */
+  public string $k_business_type = '0';

```

Comment:
The SDK is intended to work with a range of PHP versions (5.5, 7.X and 8+) as such we don't type hint properties as that break compatibility with earlier PHP versions.

Reply:
Fixed